common/lib/protocol-definitions/src/tokens.ts

```diff
@@ -66,23 +66,52 @@ export interface ITokenClaims {
     jti?: string;
 }
 
+/**
+ * TODO
+ */
 export interface ISummaryTokenClaims {
+    /**
+     * TODO
+     */
     sub: string;
+
+    /**
+     * TODO
+     */
     act: IActorClient;
+
+    /**
+     * TODO
+     */
     claims: ITokenClaims;
 }
 
+/**
+ * TODO
+ */
 export interface IActorClient {
+    /**
+     * TODO
+     */
     sub: string;
 }
 
 /**
  * Abstracts the discovery of claims contained within a token.
  */
 export interface ITokenService {
+    /**
+     * TODO
+     */
     extractClaims(token: string): ITokenClaims;
 }
 
+/**
+ * TODO
+ */
 export interface ITokenProvider {
```

Comment:
Is this even used anywhere?

Reply:
It is, yes. In fact the azure-client package re-exports this as a part of its own API.

Reply:
These appear to be duplicated in routerlicious-driver. @jatgarg @andre4i do either of you know if this is intentional? Should these types be de-duplicated?

Reply:
It looks like the protocol-definitions types are not being used?